content/overview/pricing/rates.md

```diff
@@ -78,6 +78,16 @@ Free of charge and without any paperwork, anyone with a government email address
 
 [Learn more about sandboxes.]({{< relref "overview/pricing/free-limited-sandbox.md" >}})
 
+## Quota pricing
+
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
```

Comment:
The quota is a reservation, not what you actually use. What's unused isn't refunded. So if you reserve, say, 5GB, your price is $16.50 per day. Repeated over a 30-day month, that's $495 per month. You pay the quota price for every single day, and then we assess and bill monthly.

Maybe it'd be better to say something like:
`Quotas are set in MB of memory you reserve. The price is $0.0033 per MB, and the price is considered per day. We assess the usage monthly, which in practice works out to about $99 per GB for a 30-day month.`

Then maybe pull all the calculation stuff over to the dedicated page for this (`quotas.md`)?